### PR TITLE
docs: Upgrade docz to v2

### PR DESCRIPTION
**Pull request by lightnet328**
# WHAT

- Upgrade docz to v2
- Create docz package in docs dir
- Remove an unused config

# WHY

- The docz package is for using "Shadowing`
  - https://www.gatsbyjs.org/docs/themes/shadowing/

localhost_3000_(iPad Pro)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -62,17 +62,20 @@
...
     "cross-env": "6.0.3",
     "del-cli": "3.0.0",
-    "docz": "1.3.2",
-    "docz-theme-default": "1.2.0",
+    "docz": "2.2.0",
     "enzyme": "3.11.0",
     "enzyme-adapter-react-16": "1.15.2",
     "eslint": "6.8.0",
     "eslint-config-prettier": "6.9.0",
+    "eslint-loader": "3.0.3",
```
Comment (lightnet328) [inline package.json]:
```
warning “docz > gatsby-plugin-eslint@2.0.8” has unmet peer dependency “eslint-loader@^3.0.2".
```
For resolving this.

**Comment by lightnet328** [comment]
Thanks for your review! I fixed issues caused by created a new package for docz.

**Comment by rot1024** [comment]
Thank you for great work! But I want you to fix more following points:

## 1. Remove extra heading on component pages

> name: Viewer route: /components/Viewer
> 
> menu: Components

You can fix `docs/scripts/generateDoc.js` to achieve it.

image

## 2. Tweak styles of tables

Adjust the styles of the table to be closer to the previous document.

image

to

image

## 3. Add background color to `<code>` block, and tweak text font styles

image

to

image

**Comment by lightnet328** [comment]
Thanks for reviewing! I've just changed. Please review it again.

- Table and InlineCode styles
  - imported from current styles and adjusted it for light mode
- Fix generateDoc.js

**Comment by rot1024** [comment]
Excellent. Thanks to you, the appearance of the document has become more beautiful.

Sadly, the following issue has left:

Storybook on docz does not work because `/cesium/Widgets/widgets.css` and `/cesium/Cesium.js` are not found.

Please change webpack config of docz to copy Cesium files and set env var on build.

Reference:

```js
  modifyBundlerConfig: config => ({
    ...config,
    // for storybook
    externals: {
      ...config.externals,
      cesium: "Cesium",
    },
    // for storybook
    plugins: [
      ...config.plugins,
      new webpack.DefinePlugin({
        CESIUM_BASE_URL: JSON.stringify("/cesium"),
      }),
      new CopyPlugin([
        {
          from: "node_modules/cesium/Build/Cesium",
          to: "cesium",
        },
      ]),
    ],
  }),
```

**Comment by lightnet328** [comment]
I’m so sorry to bother you many times. I added `gatsby-node.js` to copy `cesium` modules.

File: package.json [hunk 2]
```diff
@@ -13,24 +13,21 @@
...
+    "storybook:build": "build-storybook -s .storybook/public -o docs/.docz/public/examples",
+    "doc": "yarn --cwd docs dev",
+    "doc:build": "yarn --cwd docs build && build-storybook -o docs/.docz/public/examples",
+    "doc:generate": "yarn --cwd docs generate"
   },
-  "files": [
-    "src",
-    "dist"
-  ],
+  "files": ["src"],
```
Comment (rot1024) [inline package.json]:
```suggestion
  "files": [
    "src",
    "dist"
  ],
```

**Pull request merged**